native/general_double.go

```diff
@@ -10,6 +10,21 @@ import (
 	"math"
 )
 
+func checkMatrix(m, n int, a []float64, lda int) {
```

Comment:
I tried that, but it doesn't work. The functions are copied as is with the given name. Where is the source code for single_precision?